tests/test_wps_convolution.py

```diff
@@ -73,14 +97,61 @@ def test_wps_convolution(
     config_file,
     config_dict,
 ):
-    params = (
-        f"case_id={case_id};"
-        f"run_startdate={run_startdate};"
-        f"stop_date={stop_date};"
-        f"domain=@xlink:href={domain};"
-        f"param_file=@xlink:href={param_file};"
-        f"input_forcings=@xlink:href={input_forcings};"
-        f"config_file=@xlink:href={config_file};"
-        f"config_dict={config_dict};"
+    params = build_params(
+        case_id,
+        run_startdate,
+        stop_date,
+        domain,
+        param_file,
+        input_forcings,
+        config_file,
+        config_dict,
     )
     run_wps_process(Convolution(), params)
+
+
+@mark.slow
+@mark.online
+@mark.parametrize(
+    (
+        "case_id",
+        "run_startdate",
+        "stop_date",
+        "domain",
+        "param_file",
+        "input_forcings",
+        "config_dict",
+    ),
+    [
+        (
+            "sample",
+            None,  # Date cannot be None
```

Comment:
So in the case that we are testing a particular "bad" item I think we should consider giving the test method a name related to that particular error. In this case something like `test_wps_convolution_bad_date`. If this test were to break a developer would be able to know exactly which part is not working, rather than having to investigate comments in the parameters.